lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -9,52 +9,14 @@ function canMultiplyLikeTermConstantNodes(node) {
     return false;
   }
   const args = node.args;
-  if (!args.every(n => isConstantOrConstantPower(n))) {
+  if (!args.every(n => ConstantOrPowerTerm.isConstantOrConstantPower(n))) {
     return false;
   }
-  const constantTermList = node.map(n => getBaseNode(n));
-  const firstTerm = constantTermList.args[0];
-  const restTerms = constantTermList.args.slice(1);
+  const constantTermList = node.args.map(n => ConstantOrPowerTerm.getBaseNode(n));
```

Comment:
curious - what kind of tests would have failed before you fixed this? or does this code do the same thing?

if this would have broken something before, see if you can add at test that only passes with the fixed code